metaci/build/models.py

```diff
@@ -425,20 +447,18 @@ def run(self):
         self.flush_log()
 
         if (
-            self.org and self.org.name and self.org.name.lower() == "packaging"
-        ):  # Calling for any actions taken against packaging org
+            self.plan.change_traffic_control
+        ):
```

Comment:
```
if self.plan.change_traffic_control:
```

Reply:
Also this block isn't quite right. It needs to NOT return, because we still want the build status to get set to `success` at the end of the function. And it needs to go before `self.flush_log()` to make sure the log gets written.